how-to/running.rst

```diff
@@ -24,20 +24,10 @@ For the host system, any reasonably modern Ubuntu release should work.
 Other Linux distributions that have LXD should work too, though we don't
 test on them.
 
-For the containers, these instructions should work with Ubuntu 16.04 LTS and
-20.04 LTS.  We currently test on 16.04, 18.04, and 20.04, with the aim of
-upgrading production to 20.04 soon.  22.04 is known not to work yet.
+For the containers, these instructions should work with Ubuntu 20.04 LTS.
+22.04 is known not to work yet.
 
-.. note::
-   Ubuntu 22.04 ships with cgroups v2 enabled by default. Since the 16.04
-   release is too old to support it, the container doesn't get set up properly
-   as per the profile and isn't usable for setting up Launchpad. So to re-enable
-   cgroups v1, set the `systemd.unified_cgroup_hierarchy=0` kernel parameter.
-   For GRUB, this can be done by editing `/etc/default/grub` and
-   appending the parameter to the `GRUB_CMDLINE_LINUX_DEFAULT` value,
-   running `update-grub` after that, and rebooting.
-
-Launchpad requires at least Python 3.5 (i.e. newer than Ubuntu 14.04 LTS).
+Launchpad requires at least Python 3.8 (i.e. at least Ubuntu 20.04 LTS).
```

Comment:
Do we really need that rather vague "should work with Ubuntu 20.04 LTS"? That's confusing.

Could we just write something like

> Launchpad requires Ubuntu 20.04 LTS with Python 3.8. Ubuntu 22.04 is known not to work yet.

Reply:
I don't object, but I'm not going to have time to do this myself.  Please could you make this look the way you want and land it?  I think I have GitHub permissions set up so that you can edit this branch.

Reply:
Done so.